dev-support/docker-images/jupyter/Dockerfile

```diff
@@ -94,21 +94,20 @@ RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda3-py37_${MINICONDA
     conda clean --all -f -y && \
     rm -rf /home/$NB_USER/.cache/yarn
 
+RUN conda init bash 
 # Install latest sumbarine python sdk and notebook
-RUN git clone https://github.com/apache/submarine && \
-    pip install -e submarine/submarine-sdk/pysubmarine[tf,pytorch] && \
+RUN source ~/.bashrc && conda activate && \
+    git clone https://github.com/apache/submarine && \
+    pip install submarine/submarine-sdk/pysubmarine[tf,pytorch] && \
     conda install nodejs && \
     conda install -c conda-forge jupyterlab jupyterlab-git && \
-    jupyter lab build
+    jupyter lab build 
 
 # Add DeepFM example into notebook
 RUN cp submarine/submarine-sdk/pysubmarine/example/submarine_experiment_sdk.ipynb $HOME && \
     cp -r submarine/submarine-sdk/pysubmarine/example/{data,deepfm_example.ipynb,deepfm.json} $HOME && \
     rm submarine -rf
 
-# Install mlflow & minio dependencies
-RUN pip install boto3==1.17.58 mlflow==1.15.0
-
 # Install latest stable qlib
```

Comment:
@pingsutw 
I think qlib can be removed.
What do you think?

Reply:
We use `qlib` in the notebook example.
https://github.com/apache/submarine/blob/6265879ed6482fafcbc278d82ef6416b61377407/dev-support/docker-images/jupyter/Dockerfile#L117